packages/eslint-config-shakacode/rules/javascript.yml

```diff
@@ -17,3 +17,18 @@ rules:
     - 2
     - args: "all"
       argsIgnorePattern: ^_
+
+  # http://eslint.org/docs/rules/max-len
+  # https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L128
+  # Almost identical to airbnb's version, but adds ignore of `import` statements
+  max-len:
+    - 2
+    -
+      code: 100
+      tabWidth: 2
+      ignoreUrls: true
+      ignoreComments: false
+      ignoreRegExpLiterals: true
+      ignoreStrings: true
+      ignoreTemplateLiterals: true
+      ignorePattern: ^import\\s+
```

Comment:
TBH I find the upper imports more readable than the lower ones.

![screen shot 2016-11-09 at 01 09 03](https://cloud.githubusercontent.com/assets/4244251/20109051/5ce46494-a619-11e6-9d7c-9b641758a2bf.png)